glean-core/src/lib.rs

```diff
@@ -140,6 +142,7 @@ pub struct Configuration {
 #[derive(Debug)]
 pub struct Glean {
     upload_enabled: bool,
+    upload_manager: PingUploadManager,
```

Comment:
Since this is a partial implementation, do we really want to enable it here?

Reply:
Removed it in 67ccda3. I tried putting it behind a feature flag, but honestly deleting it is nicer and simpler for this situation.